src/main/java/com/bugsnag/Bugsnag.java

```diff
@@ -42,6 +42,44 @@ public void rejectedExecution(Runnable runnable, ThreadPoolExecutor executor) {
      * Initialize a Bugsnag client and automatically send uncaught exceptions.
      *
      * @param apiKey your Bugsnag API key from your Bugsnag dashboard
+     * @return The Bugsnag client instance
+     */
+    public static Bugsnag createBugsnag(String apiKey) {
+        return createBugsnag(apiKey, true);
+    }
+
+    /**
+     * Initialize a Bugsnag client.
+     *
+     * @param apiKey                 your Bugsnag API key
+     * @param sendUncaughtExceptions should we send uncaught exceptions to Bugsnag
+     * @return The Bugsnag client instance
+     */
+    public static Bugsnag createBugsnag(String apiKey, boolean sendUncaughtExceptions) {
+
+        // Look for an existing instance of Bugsnag in the appender
+        if (BugsnagAppender.getInstance() != null
+                && BugsnagAppender.getInstance().getBugsnag() != null) {
+
+            // Check that the API key matches
+            if (apiKey.equals(BugsnagAppender.getInstance().getBugsnag().config.apiKey)) {
```

Comment:
We need to validate the API key with a null check as per the constructor

Reply:
added